go/stellar/relays/relays.go

```diff
@@ -141,7 +141,7 @@ func encrypt(relay stellar1.RelayContents, encryptFor keybase1.TeamApplicationKe
 		return res, err
 	}
 	secbox := secretbox.Seal(
-		nil, clearpack[:], &nonce, (*[32]byte)(&encryptFor.Key))
+		nil, clearpack, &nonce, (*[32]byte)(&encryptFor.Key))
```

Comment:
did you use some tool or just an eagle eye for these?

Reply:
some linter found it...i can't remember which one, i was trying out a bunch.